src/Query/Builder.php

```diff
@@ -580,49 +622,47 @@ public function insert(array $values)
 
         $result = $this->collection->insertMany($values, $options);
 
-        return 1 == (int) $result->isAcknowledged();
+        return (int) $result->isAcknowledged() === 1;
     }
 
-    /**
-     * @inheritdoc
-     */
+    /** @inheritdoc */
     public function insertGetId(array $values, $sequence = null)
     {
         $options = $this->inheritConnectionOptions();
 
         $result = $this->collection->insertOne($values, $options);
 
-        if (1 == (int) $result->isAcknowledged()) {
-            if ($sequence === null) {
-                $sequence = '_id';
-            }
+        if ((int) $result->isAcknowledged() !== 1) {
```

Comment:
No need to cast the bool to an int:

```suggestion
        if (! $result->isAcknowledged()) {
```